google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProviderTest.java

```diff
@@ -795,11 +800,65 @@ public void testFromSpace() throws Exception {
     assertThat(path4.toString()).isEqualTo("/with/a%20percent");
   }
 
+  // port of test from
+  // https://github.com/broadinstitute/cromwell/pull/6491/files#diff-758dbbe823e71cc26fee7bc89cd5c434dfb76e604d51005b8327db59aab96068R300-R336
+  @Test
+  public void cromwell6491() throws Exception {
```

Comment:
Do you want to rename the test before merging?

Reply:
+1, i agree with linking to the original but i don't think it's the right name for a test

Reply:
done, new name `ensureMultipleInstancesDoNotCorruptCredentials`